ghmirror/core/mirror_requests.py

```diff
@@ -57,11 +58,11 @@ def conditional_request(method, url, auth, data=None):
         resp.headers['X-Cache'] = 'MISS'
         return resp
 
-    cache_key = (url, auth_sha)
+    cache_key = pickle.dumps((url, auth_sha))
```

Comment:
Why are you pickling here? This module does not need to care about the underlying implementation details.

Reply:
i'm with you. i don't like it either.